packages/tpetra/core/ext/TpetraExt_MatrixMatrix_ExtraKernels_def.hpp

```diff
@@ -119,7 +119,7 @@ void mult_A_B_newmatrix_LowThreadGustavsonKernel(CrsMatrixStruct<Scalar, LocalOr
   typedef typename KCRS::values_type::non_const_type scalar_view_t;
 
   // Unmanaged versions of the above
-  typedef UnmanagedView<lno_view_t> u_lno_view_t;
+  //typedef UnmanagedView<lno_view_t> u_lno_view_t; // unused
```

Comment:
Eventually we should really just get rid of this if it is unused...

Reply:
@lucbv I agree, but I'm a little shy to delete this, since I didn't write it.

Reply:
Also, this is totally _not_ the main point of this PR.  The main point is fixing #4827.

Reply:
In general, though, I agree that we shouldn't be shy about deleting code like this.